script/public/javascripts/examples.js

```diff
@@ -0,0 +1 @@
+GrainRunner('examples/conditionals.wasm').then(res => console.log(res));
```

Comment:
Why `console.log`?

Reply:
This is just an example file. `GrainRunner` returns a promise that resolves to whatever the Grain program returns. So if you want to see it, you need to print it or do something else with it, if you didn't already in your program.

Reply:
Oh, this is an example _for how to use the JS API_. Got it. I misunderstood.